int32 utcTime
int32 longitude
int32 lat
int32 elevation
